staging/src/k8s.io/apiserver/pkg/admission/plugin/policy/validating/metrics/metrics.go

```diff
@@ -29,16 +29,31 @@ const (
 	metricsSubsystem = "validating_admission_policy"
 )
 
+// ValidationErrorType defines different error types that happen to a validation expression
+type ValidationErrorType string
+
+const (
+	// ValidationCompileError indicates that the expression fails to compile.
+	ValidationCompileError ValidationErrorType = "compile_error"
```

Comment:
Does it make sense to move these also to staging/src/k8s.io/apiserver/pkg/cel/errors.go ?

Reply:
I added into here since it is only used by metrics. I could add it into metrics/errors.go in my next metric pr is that makes sense :)